import-export-cli/integration/config.yaml

```diff
@@ -9,6 +9,6 @@ environments:
 - name: production
   host: localhost
   offset: 1
-dcr-version: v0.16
+dcr-version: v0.17
 rest-api-version: v1
-apictl-version: 3.1.0
\ No newline at end of file
+apictl-version: 3.2.0
```

Comment:
add newline at the end

Reply:
Fixed via https://github.com/wso2/product-apim-tooling/pull/593